src/lib/components/UploadBtn.svelte

```diff
@@ -3,6 +3,7 @@
 
 	export let classNames = "";
 	export let files: File[];
+	export let mimeTypes: string[] = ["*/*"];
```

Comment:
```suggestion
	export let mimeTypes: string[];
```

`mimeTypes` prop always has valid value. No ?